src/runtime/smart-button-api.js

```diff
@@ -93,12 +93,7 @@ export class SmartSubscriptionButtonApi {
    */
   handleSmartBoxClick_(smartBoxMessage) {
     if (smartBoxMessage && smartBoxMessage.getIsClicked()) {
-      if (!this.callback_) {
```

Comment:
I think this check is unnecessary. The only instantiator for the Smart Button always defines a callback